Library/PTPusher.m

```diff
@@ -375,7 +376,7 @@ - (void)pusherConnection:(PTPusherConnection *)connection didDisconnectWithCode:
 {
   NSError *error = nil;
 
-  if (errorCode > 0) {
+  if (errorCode > 0 && errorCode != SRStatusCodeNormal && errorCode != SRStatusCodeGoingAway) {
```

Comment:
`SRWebSocket` will return at least these two codes on disconnect, Pusher wasn't respecting these codes and continued to try to reconnect anyway.